src/js/store/projects/reducer.ts

```diff
@@ -0,0 +1,4 @@
+export interface Project {
+  name: string;
+  description: string;
+}
```

Comment:
For each product, we’ll want a list of `projects` and also a `next` url (maybe also a `notFound` list, but that can wait for now). We _could_ store that data on individual products, but I wonder if it might be cleaner to have a separate `projects` top-level state object, but then have it separated by product (probably `id`?)…

```
{
  products: ...,
  projects: {
    [product.id]: {
      projects: [...],
      next: null,
    },
    ...
  }
}
```